website/docs/d/me_installation_template.html.markdown

```diff
@@ -22,24 +22,20 @@ data "ovh_me_installation_template" "mytemplate" {
 
 The following attributes are exported:
 
-* `available_languages`: List of all language available for this template.
-* `beta`: This distribution is new and, although tested and functional, may still display odd behaviour.
+* `available_languages`: Deprecated, will be removed in next release.
```

Comment:
you should keep the descriptions and maybe add this disclaimer at the end of the line